### PR TITLE
Update Icy Conditions

### PR DESCRIPTION
**Pull request by danfox01**
Updated content for note and set shouldDisableForm: true. Validated JSON before opening PR.

This content change will allow us to confirm new functionality in prod. Additional JSON changes to follow after release.

### CODE REVIEW

File: data/production/categories.json [hunk 1]
```diff
@@ -193,7 +193,8 @@
 			{
 				"name": "Icy conditions",
 				"id": 1011069,
```
Comment (martypowell) [inline data/production/categories.json:195]: I think the ID Should be updated to "D" something?

Comment (danfox01) [reply data/production/categories.json:195]: This is a weird one. Icy conditions is in citysourced, they just don’t want it to be submitted. So it has a legit citysourced id and doesn’t need a dummy. My thinking is only use dummies when there’s not an id, regardless of what we’re doing with disabling the form. What do you think?

Comment (martypowell) [reply data/production/categories.json:195]: as long as we test this in dev, and it works as expected, I'm okay with this. Although it is a little confusing.

Comment (danfox01) [reply data/production/categories.json:195]: Ok cool. Yeah it’s tested and works. I might change the approach when we add more items but for now this is what makes most sense to me.

**Review by martypowell — APPROVED** [review]

**Pull request merged**